Cargo.toml

```diff
@@ -51,6 +51,7 @@ thiserror = "1.0.20"
 once_cell = "1.5.0"
 never = "0.1.0"
 chrono = "0.4.19"
+either = "1.6.1"
```

Comment:
Maybe just `1.6` would suffice?

Reply:
I think it would, but we use precise versions everywhere in the `Cargo.toml`, so maybe it's a topic for a different PR?

Reply:
It seems that libraries should use `x.y` instead of `x.y.z` except for the real need for bug fixes.
Yes, a topic for another PR.